Composer/packages/client/src/constants.ts

```diff
@@ -379,3 +390,25 @@ export const triggerNotSupportedWarning = () =>
   formatMessage(
     'This trigger type is not supported by the RegEx recognizer. To ensure this trigger is fired, change the recognizer type.'
   );
+
+export const feedDictionary: { [key in FeedName]: string } = {
+  firstPartyCsharp:
+    'https://registry.npmjs.org/-/v1/search?text=conversationalcore&size=100&from=0&quality=0.65&popularity=0.98&maintenance=0.5',
+  firstPartyNode:
```

Comment:
we'll want to remove/disable the JS feed before shipping

Reply:
Handled